src/lxc/cgroups/cgfsng.c

```diff
@@ -2209,15 +2213,12 @@ __cgfsng_ops static int cgfsng_get(struct cgroup_ops *ops, const char *filename,
 				     char *value, size_t len, const char *name,
 				     const char *lxcpath)
 {
-	int ret = -1;
-	size_t controller_len;
-	char *controller, *p, *path;
+	__attribute__((__cleanup__(__auto_free__))) char *controller;
```

Comment:
Should we add a macro to shorten this to memory_utils.h? `lxc_autofree`?

Reply:
done